packages/govern/public/proposal.ts

```diff
@@ -65,14 +66,14 @@ export type PayloadType = {
   submitter: string
   executor: string
   actions: ActionType[]
-  allowFailuresMap: BytesLike
-  proof: BytesLike
+  allowFailuresMap: utils.BytesLike
+  proof: utils.BytesLike
 }
 
 export type ActionType = {
   to: string
   value: BigNumberish
-  data: BytesLike
+  data: utils.BytesLike
```

Comment:
@yuetloo why does `BytesLike` come from `utils` ? and `BigNumberish` doesn't come from it ? there're many places this happens.